src/main/java/com/example/learninghub/problem/ProblemController.java

```diff
@@ -23,4 +24,9 @@ public ProblemController(ProblemService problemService) {
     public String get() {
```

Comment:
Same as above

Reply:
Deleted in [d1998f1](https://github.com/Lukasz-gro/learning-hub-server/pull/10/commits/d1998f168ed5e3cb7ea96de92db7e855ea9ba3b7)